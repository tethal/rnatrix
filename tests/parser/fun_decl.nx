fun f1() {
}

fun f2(arg1) {
}

fun f3(arg1, arg2) {
}
