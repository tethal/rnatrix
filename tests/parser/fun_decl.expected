Program @<string>:9:1-1
  FunDecl("f1") @<string>:1:5-7
  FunDecl("f2") @<string>:4:5-7
    Param("arg1") @<string>:4:8-12
  FunDecl("f3") @<string>:7:5-7
    Param("arg1") @<string>:7:8-12
    Param("arg2") @<string>:7:14-18
